README.md

```diff
@@ -1,5 +1,16 @@
-# docs-community
+# Python Docs WG
 
 [![Documentation Status](https://readthedocs.org/projects/docs-community/badge/?version=latest)](https://docs-community.readthedocs.io/en/latest/?badge=latest)
- 
-Community management for documentation contributors and the Docs Workgroup
+
+The PSF Documentation Working Group (Docs WG) was chartered by the PSF to help bootstrap a larger Python documentation team and community.
+This repo serves as documentation for the Docs WG, which is funny, in a recursive kind of way.
+
+For example, to find out more about us and what we do, [read the docs](https://docs-community.readthedocs.io/en/latest/).
```

Comment:
I'm not sure how "finding out more about us" is an "example"? I find this rather confusing, which is likely more so for many non-native speakers—I've found out firsthand that humor, irony and cultural references don't always translate well, and style guides for docs (including my own) tend to proscribe it for that reason.